mkt/extensions/views.py

```diff
@@ -189,6 +189,14 @@ class ReviewersExtensionViewSet(CORSMixin, SlugOrIdMixin, MarketplaceView,
     queryset = Extension.objects.without_deleted().pending()
     serializer_class = ExtensionSerializer
 
+    @list_route(queryset=Extension.objects.without_deleted().public()
+                                          .pending_with_versions())
+    def updates(self, *args, **kwargs):
```

Comment:
for posterity: I wonder if you could just have called `self.list()` here, since you are overriding `queryset`. Would be neat if it worked.